### PR TITLE
optimize restructure

### PR DESCRIPTION
**Pull request by christinaausley**
Minor restructuring of lengthy Optimize sections for user-friendliness.

@HanselIdes how do you like this organization of the docs? Is there anywhere else you feel we could consolidate/break longer docs into more manageable docs?

Additionally, are there any docs that need a refresh based on your latest updates, like Alerts?

### CODE REVIEW

File: docs/components/optimize/userguide/process-analysis/report-analysis/edit-mode.md [hunk 1]
```diff
@@ -0,0 +1,369 @@
...
+5. User task
+  * Count: View how often each user task has been executed.
+  * Duration: View how long each user task took to complete.
+6. Variable: View an aggregation of values for a specific numeric variable of the process definition.
+
+It is possible to display both count and duration measures for a single view in the same report.
+
+Subsequently, you need to define how to group the data. Think of it as applying a metric to your input, where you break up the data by date, flow nodes, variable values, or other properties. For that, you have different options:
+
+- None: Do not split up the data.
```
Comment (HanselIdes) [inline docs/components/optimize/userguide/process-analysis/report-analysis/edit-mode.md:68]: I recommend formatting these options (none, flow nodes...) so they stand out more

File: docs/components/optimize/userguide/process-analysis/report-analysis/edit-mode.md [hunk 2]
```diff
@@ -0,0 +1,369 @@
...
+
+![Duration Aggregation Selection](./img/durationAggregation.png)
+
+Reports with multiple aggregations that have a [second "Group by"](#reports-with-a-second-group-by-option) can only be visualized as table.
+
+### User task duration time
+
+In user task duration reports, you have the opportunity to select which part of the user task's lifecycle you want to see in the report:
+
+- Idle: View how long each user task was considered idle (not claimed by an assignee/user) during its execution.
```
Comment (HanselIdes) [inline docs/components/optimize/userguide/process-analysis/report-analysis/edit-mode.md:157]: Idle vs work is C7 only

**Review by HanselIdes — APPROVED** [review]
I have a few minor comments. I didn't provide a thorough, line-by line review because I don't get much feedback about the readability and organization of our docs.

File: docs/components/optimize/userguide/additional-features/alerts.md [hunk 3]
```diff
@@ -4,13 +4,11 @@ title: Alerts
 description: Get a notification as soon as your system is behaving in an unexpected manner.
 ---
 
-<span class="badge badge--platform">Camunda Platform 7 only</span>
-
 :::note
-You must configure the email service to receive notifications. See the [technical guide](../../../../self-managed/optimize-deployment/setup/installation.md) for which properties need to be defined.
+For Camunda Platform 7, you must configure the email service to receive notifications. See the [technical guide](../../../../self-managed/optimize-deployment/setup/installation.md) for which properties need to be defined.
```
[the remark this answers is not in the document]
Comment (christinaausley) [reply docs/components/optimize/userguide/additional-features/alerts.md:8]: So this note is applicable to both Platform and Cloud or should we keep that this is currently only applicable to Platform?

File: docs/components/optimize/userguide/additional-features/alerts.md [hunk 4]
```diff
@@ -4,13 +4,11 @@ title: Alerts
...
 :::
 
-Optimize's alerting functionality can be used to notify you when your report hits a predefined critical value. You can create alerts for any number reports that exist within a collection. Inside the collection, navigate to the **Alerts** tab to see all alerts defined for reports in this collection. You can manage an alert by moving the mouse over the alert entry and clicking the **Edit** or **Delete** buttons in the context menu on the right side of the page.
+Optimize's alerting functionality can be used to notify you when your report hits a predefined critical value. You can create alerts for any number reports that exist within a collection. Inside the collection, navigate to the **Alerts** tab to create and view all alerts defined for reports in this collection. You can manage an alert by moving the mouse over the alert entry and clicking the **Edit** or **Delete** buttons in the context menu on the right side of the page.
```
[the remark this answers is not in the document]
Comment (christinaausley) [reply docs/components/optimize/userguide/additional-features/alerts.md:11]: We should likely update the following screenshot as well then. Who is the best contact within the Optimize team for screenshot automation to make this update?

File: docs/components/optimize/userguide/process-analysis/report-analysis/edit-mode.md (hunk 1 above)
Comment (christinaausley) [reply docs/components/optimize/userguide/process-analysis/report-analysis/edit-mode.md:68]: Adjusted in latest commit

File: docs/components/optimize/userguide/process-analysis/report-analysis/edit-mode.md (hunk 2 above)
Comment (christinaausley) [reply docs/components/optimize/userguide/process-analysis/report-analysis/edit-mode.md:157]: Noted in latest commit

**Comment by christinaausley** [comment]
@HanselIdes Thanks for your quick review! I made a few comments in response -- let me know your thoughts. 

If anything changes regarding feedback about the readability and organization of this Optimize section (now that so much information has been migrated over), do let me know.

File: docs/components/optimize/userguide/additional-features/alerts.md (hunk 3 above)
[the remark this answers is not in the document]
Comment (HanselIdes) [reply docs/components/optimize/userguide/additional-features/alerts.md]: I would remove the Platform Only tag on Alerts and then mention that Webhooks are Platform-only in an appropriate place

File: docs/components/optimize/userguide/additional-features/alerts.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (HanselIdes) [reply docs/components/optimize/userguide/additional-features/alerts.md:11]: I'd start with @RomanJRW and he can direct you to someone else, as needed

[the remark this answers is not in the document]
Comment (christinaausley) [reply docs/components/optimize/userguide/additional-features/alerts.md:11]: @HanselIdes Will open a new PR for these Alert-specific screenshots/text changes.

**Comment by christinaausley** [comment]
@akeller Mind taking a quick peak at these changes? Can make any urgent fixes today and get this merged.

**Review by akeller — APPROVED** [review]
No concerns on my side for the scope of this PR.

I am missing some linking across these files though. This isn't urgent, so opening a PR in the future is fine. For now, I will create issues with the info below.

From `userguide/collections-dashboards-reports/` inline links to
* Creating dashboards
* Creating reports

Linking to and from the guide (`guides/improve-processes-with-optimize/`) into the rest of the Optimize content, particularly around creating reports and dashboards. Additional resources should also include links to Optimize content on docs.camunda.io. Other links are fine, although the 3.6 release link should be removed since 3.6 docs are available on docs.camunda.org.

Another item, though maybe not a PR or issue-worthy topic - **Edit mode** is a huge file. I'm not sure if there is action we can take in docs alone, but I'd love to collaborate with the Optimize team, including product design, to have a more intuitive experience requiring fewer docs sections. 

Separate from linking and likely another PR - I wasn't sure if a **Combined report** was a different kind of report. From the screenshots, it looks like there are 3 options - Process Report, Combined Process Report, and Decision Report. Is this a "Combined Report" or a "Combined Process Report" or does the name not really matter here and can be used interchangeably? (This is likely a product management question for Eric.) Does this mean Process Reports and Decision Reports are both described in the single report section of **Creating reports**?

**Pull request merged**